### PR TITLE
Set up initial CLI command and configuration

### PR DESCRIPTION
**Pull request by hakbailey**
### What does this PR do?

Updates the app name, command structure, and config from the template repo defaults.

### How can a reviewer manually see the effects of these changes?

Run the command locally and confirm it prints some output.

### Includes new or updated dependencies?

YES

### What are the relevant tickets?

- https://mitlibraries.atlassian.net/browse/IN-714

### Developer

- [x] All new ENV is documented in README (or there is none)
- [x] Stakeholder approval has been confirmed (or is not needed)

### Code Reviewer

- [ ] The commit message is clear and follows our guidelines (not just this pull request message)
- [ ] There are appropriate tests covering any new functionality
- [ ] The documentation has been updated or is unnecessary
- [ ] The changes have been verified
- [ ] New dependencies are appropriate or there were no changes

### CODE REVIEW

**Comment by hakbailey** [comment]
Note: ignore "failed" dev container deploy, it's actually working fine and we'll resolve the error message outside this PR.

File: Dockerfile [hunk 1]
```diff
@@ -9,4 +9,4 @@ RUN apt-get update && apt-get upgrade -y && apt-get install -y git
 COPY Pipfile* /
 RUN pipenv install
 
-ENTRYPOINT ["pipenv", "run", "my_app"]
+ENTRYPOINT ["pipenv", "run", "cc-slips"]
```
Comment (ehanson8) [inline Dockerfile:12]: Totally optional suggestion: we're juggling `cc-slips`, `cc_slips`, `credit-card-slips` across the repo, maybe just simplify to `slips`?

File: tests/test_cli.py [hunk 2]
```diff
@@ -1,17 +1,40 @@
...
+@freeze_time("2023-01-02")
+def test_cli_only_required_options(caplog, runner):
+    result = runner.invoke(
+        main, ["-s", "source@example.com", "-r", "recipient@example.com"]
+    )
     assert result.exit_code == 0
     assert "Logger 'root' configured with level=INFO" in caplog.text
-    assert "Running process" in caplog.text
+    assert "Starting credit card slips process" in caplog.text
     assert "Total time to complete process" in caplog.text
```
Comment (ehanson8) [inline tests/test_cli.py:14]: Do we want to check for `Command called with...` log entry here as well for consistency?

**Review by ehanson8 — APPROVED** [review]
Looks good, 2 optional inline suggestions

File: tests/test_cli.py (hunk 2 above)
Comment (hakbailey) [reply tests/test_cli.py:14]: Nope, that gets logged at debug level so it won't show up in the logs here.

**Pull request merged**